components/global_search/child_components/search_option/SearchOption.js

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react';
+
+class SearchOption extends Component {
+
+    render_search_options(arr) {
+        return arr.map((item, index) =>
+            <div key={index}>
+                <input name={this.props.group_name} type="radio" id={item.id}
```

Comment:
Hi @AshTNA - @punalsc suggested that you could destructure `props` here (if you wanted to).